website/source/docs/operating-a-job/resource-utilization.html.md

```diff
@@ -41,7 +41,6 @@ $ nomad alloc-status -stats 04d9627d
 And here is some sample output:
 
 ```text
```

Comment:
I think it would be better to remove Line 41 and merge the two outputs.

Reply:
I would not agree, since having the command separated from the output is how it's done everywhere else, e.g.:
https://github.com/hashicorp/nomad/blob/master/website/source/docs/operating-a-job/inspecting-state.html.md

Reply:
Would you mind unifying them on that page to combine the command and sample output. Thank you!

Reply:
I made this change so we could get this fix merged in, just because this started to idle.